src/jcstress/java/reactor/pool/SimpleDequePoolStressTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2018-Present Pivotal Software Inc, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.pool;
+
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.openjdk.jcstress.annotations.Actor;
+import org.openjdk.jcstress.annotations.Arbiter;
+import org.openjdk.jcstress.annotations.JCStressTest;
+import org.openjdk.jcstress.annotations.Outcome;
+import org.openjdk.jcstress.annotations.State;
+import org.openjdk.jcstress.infra.results.LI_Result;
+
+import reactor.core.publisher.Mono;
+
+import static org.openjdk.jcstress.annotations.Expect.ACCEPTABLE;
+import static org.openjdk.jcstress.annotations.Expect.FORBIDDEN;
+
+public class SimpleDequePoolStressTest {
+
+	@JCStressTest
+	@Outcome(id = {"ACQUIRED '200', 0"}, expect = ACCEPTABLE,  desc = "acquired")
+	@Outcome(id = {"NOT ACQUIRED, 1"}, expect = ACCEPTABLE,  desc = "acquired failed fast during/before destroy")
+	@Outcome(id = {"ACQUIRED '1', 0"}, expect = FORBIDDEN,  desc = "acquired the first resource, before destroy")
+	@Outcome(id = {"ACQUIRED '1234', 0"}, expect = FORBIDDEN,  desc = "acquired the first resource, after destroy")
```

Comment:
No need for curly braces in `id`. Also see the comment about using Strings as results.